packages/jsii-pacmak/jest.config.mjs

```diff
@@ -2,4 +2,5 @@ import { overriddenConfig } from '../../jest.config.mjs';
 
 export default overriddenConfig({
   coveragePathIgnorePatterns: ['/node_modules/', '<rootDir>/test'],
+  watchPathIgnorePatterns: ['.*\\.tsx?$'],
```

Comment:
Note - this makes `jest --watch` a lot more pleasant to use, as it'd only re-run tests onces `tsc --watch` updates the `.js` files instead of possibly re-running when the `.ts` file changed (which yields unexpected outcome).

I'm contemplating moving this up to the baseline configuration but need to evaluate impact first.